app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::ItemsController < ApplicationController
+  respond_to :json, :xml
+
+  def index
+    respond_with Item.all 
+  end
+
+
```

Comment:
Let's make sure these are all inheriting from Base Controller. Also do we want base controller namespaced or just an ApiBaseController on the same level as application controller